helm/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: pagopa-notifications-service
 description: Microservice that handles notifications in pagoPA platform
 type: application
-version: 0.5.13
-appVersion: 0.5.13
+version: 0.5.0
```

Comment:
Should this version changes be rollbacked?

Reply:
yes